lib/src/main/kotlin/com/lemonappdev/konsist/api/ext/list/KoTypeProviderListExt.kt

```diff
@@ -54,10 +54,11 @@ fun <T : KoTypeProvider> List<T>.withTypeOf(kClass: KClass<*>, vararg kClasses:
  */
 fun <T : KoTypeProvider> List<T>.withoutTypeOf(kClass: KClass<*>, vararg kClasses: KClass<*>): List<T> =
```

Comment:
1. For this project, we are adding a new line before and after multiline if/else block.
2. Please simple conditions is a single line

(changes for this and another method)
```kotlin
@Deprecated("Will be removed in v1.0.0.", ReplaceWith("withoutSomeParentsOf"))
fun <T : KoParentClassProvider> List<T>.withoutParentClassOf(kClass: KClass<*>, vararg kClasses: KClass<*>): List<T> =
    filter {
        val missesAllParentClasses = if (kClasses.isNotEmpty()) {
            kClasses.none { kClass -> it.hasParentClass(kClass.simpleName) }
        } else {
            true
        }

        !it.hasParentClass(kClass.simpleName) && missesAllParentClasses
    }
```